Source/WebCore/rendering/style/RenderStyle.h

```diff
@@ -368,9 +368,11 @@ class RenderStyle {
     WEBCORE_EXPORT const FontCascade& fontCascade() const;
     WEBCORE_EXPORT const FontMetrics& metricsOfPrimaryFont() const;
     WEBCORE_EXPORT const FontCascadeDescription& fontDescription() const;
+    float fontSize() const;
```

Comment:
I think we shouldn't add this. I think it's a feature that the client has to pick whether they want the specified or computed one. The relationship between them is somewhat complicated due to iOS text autosizing.